internal/reader/home_chain.go

```diff
@@ -348,25 +366,43 @@ type ChainConfig struct {
 	Config chainconfig.ChainConfig `json:"config"`
 }
 
-// OCR3Config mirrors CCIPConfig.sol's OCR3Config struct
+// nolint: lll
```

Comment:
["machine-readable comments should have no space by Go convention"](https://golangci-lint.run/usage/false-positives/)
```suggestion
//nolint:lll // url
```